Blog-Andrei-Dona.html

```diff
@@ -0,0 +1,113 @@
+            voluptates tempora esse dignissimos
+            facilis adipisci, eveniet repudiandae aperiam, dolorum sit reprehenderit! Aliquam saepe sequi voluptatem
+            dolorum cumque. Itaque, porro?Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde, atque
+            voluptates tempora esse dignissimos
+            facilis adipisci, eveniet repudiandae aperiam, dolorum sit reprehenderit! Aliquam saepe sequi voluptatem
+            dolorum cumque. Itaque, porro?Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde, atque
+            voluptates tempora esse dignissimos
+            facilis adipisci, eveniet repudiandae aperiam, dolorum sit reprehenderit! Aliquam saepe sequi voluptatem
+            dolorum cumque. Itaque, porro?Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde, atque
+            voluptates tempora esse dignissimos
+            facilis adipisci, eveniet repudiandae aperiam, dolorum sit reprehenderit! Aliquam saepe sequi voluptatem
+            dolorum cumque. Itaque, porro?Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde, atque
+            voluptates tempora esse dignissimos
+            facilis adipisci, eveniet repudiandae aperiam, dolorum sit reprehenderit! Aliquam saepe sequi voluptatem
+            dolorum cumque. Itaque, porro?
+        </p>
+
+        <p style="position: absolute;top: 40px;right: -100px;border-radius: 15px;
+            border: solid black;height: 100px; width: 150px;font-size: 23px;text-align: center;">
+            Content 0
+            <span style="position: absolute;right: 200px;">Link</span>
+        </p>
+        <p style="position: absolute;top: 0px;right: -100px;border-radius: 15px;
+        border: solid black;height: 1000px; width: 300px;font-size: 23px;text-align: center;">
+            Side Bar
+        </p>
+        <p style="position: absolute;top: 40px;right: -100px;border-radius: 15px;
+        border: solid black;height: 100px; width: 150px;;top: 250px;font-size: 23px;text-align: center;">
+            Content 1
+            <span style="position: absolute;right: 200px;">Link</span>
+        </p>
+        <p style="position: absolute;top: 40px;right: -100px;border-radius: 15px;
+    border: solid black;height: 100px; width: 150px;top: 500px;font-size: 23px;text-align: center;">
+            Content 2
+            <span style="position: absolute;right: 200px;">Link</span>
+        </p>
+        <p style="position: absolute;top: 40px;right: -100px;border-radius: 15px;
+border: solid black;height: 100px; width: 150px;top: 750px;font-size: 23px;text-align: center;">
+            Content 3
+            <span style="position: absolute;right: 200px;">Link</span>
+        </p>
+
+    </div>
+    <div>
+        <textarea style="font-family:sans-serif;font-size:1.2em; background-color: blanchedalmond;
+        position: relative;top: 1400px;left: 200px;height: 120px;width: 700px;">

```

Comment:
Nu putem pozitiona relativ sectiunea de comentarii deoarece nu putem sti ce lungime are articolul. In acest caz tu ai pus 1400px, dar dimensiunea articolului se poate modifica oricand.
Mai incearca 😃